internal/api/status.go

```diff
@@ -24,15 +25,18 @@ func status(config *RouterConfig) API {
 		ContentType = "application/json"
 	)
 	type Response struct {
-		Version string        `json:"version"`
-		OS      string        `json:"os"`
-		Arch    string        `json:"arch"`
-		UpTime  time.Duration `json:"uptime"`
-
-		CPUs       int    `json:"num_cpu"`
-		UsableCPUs int    `json:"num_cpu_used"`
-		HeapAlloc  uint64 `json:"mem_heap_used"`
-		StackAlloc uint64 `json:"mem_stack_used"`
+		Version    string        `json:"version"`
+		OS         string        `json:"os"`
+		Arch       string        `json:"arch"`
+		UpTime     time.Duration `json:"uptime"`
+		CPUs       int           `json:"num_cpu"`
+		UsableCPUs int           `json:"num_cpu_used"`
+		HeapAlloc  uint64        `json:"mem_heap_used"`
+		StackAlloc uint64        `json:"mem_stack_used"`
+
+		KeyStoreLatency     time.Duration `json:"keystore_latency,omitempty"`
```

Comment:
Not really a fan of `time.Duration` in JSON, since it is really only specific to Go. 

But I see you already have one in there. Let's all least make it 0-able:

```suggestion
		KeyStoreLatency     time.Duration `json:"keystore_latency"`
```

A more JSON-like approach would be a float64, with unit in the title:
```
		KeyStoreLatencySecs     float64 `json:"keystore_latency_secs"`
```

(milliseconds would work as well, though we might as well go with a base unit).